tests/foreman/api/test_hostgroup.py

```diff
@@ -2,6 +2,7 @@
 """Tests for the ``hostgroups`` paths."""
 from fauxfactory import gen_string
 from nailgun import client, entities, entity_fields
+from nailgun.entity_mixins import _get_entity_id
```

Comment:
I am not comfortable calling a function prefixed with _    If this function is to be usable by other modules, please make it public.  Prefixed with _ has a reason and I dont want us to change it.  Non-blocker for this PR though